pgscatalog_utils/match/combine_matches.py

```diff
@@ -49,7 +52,20 @@ def _check_duplicate_vars(matches: pl.LazyFrame):
                                  .collect()
                                  .get_column('count')
                                  .to_list())
-    assert max_occurrence == [1], "Duplicate IDs in final matches"
+
+    match n := max_occurrence[0]:
+        case None:
+            logger.critical("No variant matches found")
+            logger.critical(
+                "Did you set the correct genome build? Did you impute your genomes?")
+            raise ValueError
+        case _ if n > 1:
+            logger.critical("Duplicate IDs in final matches")
+            logger.critical(
+                "Please double check your genomes for duplicates and try again")
+            raise ValueError
+        case _:
+            logger.info("Scoring files are valid (no duplicate variants found)")
```

Comment:
Should this not be in match_variants scrip too? Considering that script can also  be run in the full scorefile mode (although it isn't within the pipeline).

Reply:
Actually maybe it's not needed because match_variants will have already handled duplicates because they're all in one file?

Reply:
From @nebfield: